wpiutil/src/main/native/include/wpi/fs.h

```diff
@@ -28,12 +28,12 @@ using fstream = std::fstream;
 #if defined(_WIN32)
 // A Win32 HANDLE is a typedef of void*
 using file_t = void*;
+#define WPI_kInvalidFile reinterpret_cast<void*>(-1)
```

Comment:
I know the using is _right_ above this, but we should be reinterpret_casting to file_t, not void*